app.py

```
@@ -0,0 +1,148 @@
+        defined in the airdrop API.
+    db (flask_sqlalchemy.SQLAlchemy): Flask_SQLAlchemy database for State and
+        Setting tables.
+    tables (dict): Dictionary of tables in database
+"""
+
+from os.path import isfile
+import time
+from flask import Flask, Response, jsonify, request
+from flask_sqlalchemy import SQLAlchemy
+from gps import GPS
+
+app = Flask(__name__)
+app.config['SQLALCHEMY_DATABASE_URI'] = 'sqlite://'
+app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = False
+# From the flask sqlalchemy configurations page on tracking modifications:
+#   If set to True, Flask-SQLAlchemy will track modifications of objects and
+#   emit signals. The default is None, which enables tracking but issues a
+#   warning that it will be disabled by default in the future. This requires
+#   extra memory and should be disabled if not needed.
+db = SQLAlchemy(app)
+
+from state import State
+from setting import Setting
+
+db.create_all()
+
+tables = {'state': State, 'setting': Setting}
+
+
+@app.route('/v1/airdrop/<path:table>', methods=['GET'])
+def get_from_table(table):
+    """
+    Queries the table for the airdrop state/setting at a timestamp (query
+    parameter). If no timestamp is provided, the current time is used instead.
+
+    Returns:
+        flask.wrappers.Response: Response object containing a json
+            representation of the queried state/setting.
+    """
+    if table not in tables:
+        return Response('<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2 Final//EN">\n\
+            <title>404 Not Found</title>\n<h1>Not Found</h1>\n\
+            <p>The requested URL was not found on the server. \
+            If you entered the URL manually please check your spelling and try again.</p>\n',
+                        status=404, mimetype='text/html')  # Default flask 404
+
+    try:
+        timestamp = long(time.time() * 10**3) if request.args.get('timestamp') is None \
+            else long(request.args.get('timestamp'))
+    except:
```

You should include the actual exception(s) you are catching in order to prevent masking other errors that may occur